### PR TITLE
EmissLine

### PR DESCRIPTION
**Pull request by profxj**
Introduces the EmissLine class.  No significant methods yet.

Includes tests and docs

Also fixes pyyaml issue in Travis

### CODE REVIEW

**Comment by profxj** [comment]
Addresses Issue #303

**Comment by ntejos** [comment]
I left some comments; please consider them and merge.

**Comment by profxj** [comment]
I don't see your comments.  Do you?

**Comment by ntejos** [comment]
Sorry for the delay, I was able to see my comments but just realized that I did not 'submit' them according to the new github protocol.

File: docs/emissline.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.. _EmissLine:
+
+***************
+EmissLine Class
```
Comment (profxj) [inline docs/emissline.rst]: done

File: linetools/spectralline.py [hunk 2]
```diff
@@ -656,6 +678,32 @@ def __repr__(self):
...
+        """
+        #
+        self.analy.update( {
+            'datafile': '',
+            'name': self.data['name']
+            })
+
+        # Additional fundamental attributes for Emission Line
+        self.attrib.update(emiss_attrib.copy())
+
```
Comment (profxj) [inline linetools/spectralline.py]: Not sure I follow.  redshift is included as a default for SpectralLine.

**Comment by profxj** [comment]
no problem.  Back at you.  But don't merge until
@tiffanyhsyu reviews.   @tiffanyhsyu it is your turn, thanks.

**Comment by profxj** [comment]
@tiffanyhsyu indicate when you have had a chance to review it all.

**Comment by tiffanyhsyu** [comment]
I've looked through everything now! Looks good to me.

**Comment by profxj** [comment]
Merging

**Pull request merged**